examples/gno.land/p/demo/uassert/uassert.gno

```diff
@@ -416,3 +419,317 @@ func NotEmpty(t TestingT, obj interface{}, msgs ...string) bool {
 	}
 	return true
 }
+
+// Nil asserts that the specified object is nil.
+func Nil(t TestingT, obj interface{}, msgs ...string) bool {
```

Comment:
Can you document that `Nil` only checks if `obj` (as an interface) is `nil`; ie. `Nil(t, (*int)(nil)) == false`?

Reply:
same as the previous comment, I had made a minimalist implementation of this function, I added an isNil function to manage all types

https://github.com/gnolang/gno/pull/2547/commits/4077bf1665661c47b19e068ed212b3e494956f31